dashboards/grafana-dashboard-insights-rbac-operations.configmap.yaml

```diff
@@ -3906,11 +3883,12 @@ data:
       "timezone": "",
       "title": "Operations - Rbac w/CPU",
       "uid": "TjP_nMWMk",
-      "version": 8,
+      "version": 9,
       "weekStart": ""
     }
 kind: ConfigMap
 metadata:
+  creationTimestamp: null
```

Comment:
Seems this is not necessary?

Reply:
well I usually change the dashboard directly on https://grafana.stage.devshift.net and then generate the configmap using this [guide](https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/app-sre/monitoring.md#visualization-with-grafana) so that is something that is added there during that process ... and I am lazy to remove them all